config.json

```diff
@@ -39,20 +39,6 @@
           ]
         }
       ]
-    },
-    {
-      "bin": "./bin/statemint",
-      "chain": "statemint-dev",
-      "id": "300",
-      "balance": "1000000000000000000000",
-      "nodes": [
-        {
-          "wsPort": 9999,
-          "port": 31300,
-          "name": "alice",
-          "flags": ["--force-authoring", "--", "--execution=wasm"]
-        }
-      ]
```

Comment:
The statemint repo is too old for the `@polkadot/api`.

As there are no updated repo provided by parity, we can test the runtime integrations in rust, and only the polkadot integrations in javascript additionally